ReactiveCocoa/UIKit/iOS/UISearchBar.swift

```diff
@@ -0,0 +1,48 @@
+import ReactiveSwift
+import enum Result.NoError
+import UIKit
+
+private class ReactiveUISearchBarDelegate: NSObject, UISearchBarDelegate {
+
+	@objc func searchBarTextDidEndEditing(_ searchBar: UISearchBar) {}
+	@objc func searchBar(_ searchBar: UISearchBar, textDidChange searchText: String) {}
+}
+
+private var delegateKey: UInt8 = 0
+
+extension Reactive where Base: UISearchBar {
+
+	private var delegate: ReactiveUISearchBarDelegate {
+		if let delegate = objc_getAssociatedObject(base, &delegateKey) as? ReactiveUISearchBarDelegate {
+			return delegate
+		}
+
+		let delegate = ReactiveUISearchBarDelegate()
+		base.delegate = delegate
+		objc_setAssociatedObject(base, &delegateKey, delegate, .OBJC_ASSOCIATION_RETAIN_NONATOMIC)
+		return delegate
```

Comment:
Makes sense, I've pushed another commit which adds the `fatalError`. But I think the associated object is required because the `UISearchBar` only holds a `weak` reference to its `delegate`.